validator/test/fixtures/compute_disk_resource_policies/assets/data.json

```diff
@@ -0,0 +1,147 @@
+[
```

Comment:
In the test fixtures, could we include another (non-disk) asset to verify [this check](https://github.com/forseti-security/policy-library/pull/297/files#diff-fbf18d3450e452021b8e6161186a6991R33) is correct?